views/default/forms/profile/edit.php

```diff
@@ -23,24 +24,19 @@
 $profile_fields = elgg_get_config('profile_fields');
 if (is_array($profile_fields) && count($profile_fields) > 0) {
 	foreach ($profile_fields as $shortname => $valtype) {
-		$metadata = elgg_get_metadata(array(
-			'guid' => $entity->guid,
-			'metadata_name' => $shortname,
-			'limit' => false
-		));
-		if ($metadata) {
-			if (is_array($metadata)) {
-				$value = '';
-				foreach ($metadata as $md) {
-					if (!empty($value)) {
-						$value .= ', ';
-					}
-					$value .= $md->value;
-					$access_id = $md->access_id;
+
+		$annotations = $entity->getAnnotations([
```

Comment:
lets make an extra ticket for it then... as it is 1 query each profile field... some forms have a lot of them